status/statusgo_backend_new/ens.nim

```diff
@@ -9,14 +9,42 @@ proc resolver*(username: string): RpcResponse[JsonNode] {.raises: [Exception].}
 
   return core.callPrivateRPC("ens_resolver", payload)
 
+proc ownerOf*(username: string): RpcResponse[JsonNode] {.raises: [Exception].} =
+  # TODO: Use a real chain id
+  let payload = %* [1, username]
+
+  return core.callPrivateRPC("ens_ownerOf", payload)
+
 proc contentHash*(username: string): RpcResponse[JsonNode] {.raises: [Exception].} =
   # TODO: Use a real chain id
   let payload = %* [1, username]
 
   return core.callPrivateRPC("ens_contentHash", payload)
 
-proc resourceURL*(username: string): RpcResponse[JsonNode] {.raises: [Exception].} =
+proc publicKeyOf*(username: string): RpcResponse[JsonNode] {.raises: [Exception].} =
   # TODO: Use a real chain id
   let payload = %* [1, username]
-  return core.callPrivateRPC("ens_resourceURL", payload)
 
+  return core.callPrivateRPC("ens_publicKeyOf", payload)
+
+proc addressOf*(username: string): RpcResponse[JsonNode] {.raises: [Exception].} =
+  # TODO: Use a real chain id
+  let payload = %* [1, username]
+
+  return core.callPrivateRPC("ens_addressOf", payload)
+
+proc expireAt*(username: string): RpcResponse[JsonNode] {.raises: [Exception].} =
+  # TODO: Use a real chain id
+  let payload = %* [1, username]
+
+  return core.callPrivateRPC("ens_expireAt", payload)
+
+proc price*(): RpcResponse[JsonNode] {.raises: [Exception].} =
+  # TODO: Use a real chain id
+  let payload = %* [1]
+  return core.callPrivateRPC("ens_price", payload)
+
+proc resourceURL*(username: string): RpcResponse[JsonNode] {.raises: [Exception].} =
+  # TODO: Use a real chain id
```

Comment:
Will all those TODOs be addressed in this PR or another? Would they change the API of the function, since we need to pass the chainId? If so, it should be added sooner than later.

Reply:
it would be added when we finally move the app to be multichain!!! at least code is ready for that on status-go.
This is a matter of time as the new wallet will be multichain